gubled/config/config.go

```diff
@@ -35,6 +35,18 @@ var (
 	HealthEndpoint = kingpin.Flag("health-endpoint", `The health endpoint to be used by the HTTP server (value for disabling it: "")`).
 			Default(defaultHealthEndpoint).Envar("GUBLE_HEALTH_ENDPOINT").String()
 
+	Postgres = struct {
+		Host     *string
+		User     *string
+		Password *string
+		DbName   *string
+	}{
+		Host:     kingpin.Flag("postgres-host", "The postgres hostname").Default("localhost").Envar("GUBLE_POSTGRES_HOST").String(),
```

Comment:
I would not prefix all this flags with postgres..
Instead I would directly used db-host...etc